### PR TITLE
Feature/civ 1488

### PR DESCRIPTION
**Pull request by vasudevganesanhmcts**
**Before creating a pull request make sure that:**

- [*] commit messages are meaningful and follow good commit message guidelines
- [*] README and other documentation has been updated / added (if needed)
- [*] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

JIRA link (if applicable)
https://tools.hmcts.net/jira/browse/CIV-1488

Change description
Task Listener and Handler to pull the general applications in AWAITING_RESPONDENT_RESPONSE state and change their state to APPLICATION_SUBMITTED_AWAITING_JUDICIAL_DECISION if the deadline is crossed.
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by vasudevganesanhmcts** [comment]
the jenkins build passed but github is not getting the latest results. . approved after reviewing the code.  . since it was raised by me cant approve it so requesting @smitatalawadekar-hmcts  to approve on my behalf

**Review by smitatalawadekar-hmcts — APPROVED** [review]

**Pull request merged**
